tests/jerry/es.next/object-prototype-tostring.js

```diff
@@ -67,7 +67,7 @@ Map.prototype[Symbol.toStringTag] = "myStringTag5";
 assert (Map.prototype.toString.call (new Map ()) === "[object myStringTag5]");
 assert (Object.prototype.toString.call (Map) === "[object Function]");
 Map.prototype[Symbol.toStringTag] = {};
-assert (Map.prototype.toString.call (new Map) === "[object Map]");
+assert (Map.prototype.toString.call (new Map) === "[object Object]");
```

Comment:
Botf Chrome and Firefox says `Map.prototype.toString.call (new Map)` -> `[object Map]`. Why do you want to change this?

Reply:
This working perfectly, since in line `64` deletes the original `toStringTag` which is not writable and in line `65` create a new toStringTag property which becomes writable and that's why in line `70` result becomes `[object Object]`. 
If we don't delete the prototype result becomes `[object Map]`.